autobump-spring-boot/src/test/java/com/github/autobump/springboot/services/AutoBumpServiceLoggerTest.java

```diff
@@ -0,0 +1,81 @@
+package com.github.autobump.springboot.services;
+
+import com.atlassian.connect.spring.AtlassianHost;
+import com.atlassian.connect.spring.AtlassianHostRepository;
+import com.github.autobump.bitbucket.model.BitBucketGitProviderUrlHelper;
+import com.github.autobump.core.model.DependencyBumper;
+import com.github.autobump.core.model.DependencyResolver;
+import com.github.autobump.core.model.GitClient;
+import com.github.autobump.core.model.GitProvider;
+import com.github.autobump.core.model.GitProviderUrlHelper;
+import com.github.autobump.core.model.IgnoreRepository;
+import com.github.autobump.core.model.UseCaseConfiguration;
+import com.github.autobump.core.model.VersionRepository;
+import com.github.autobump.springboot.configuration.Autobumpconfig;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.system.CapturedOutput;
+import org.springframework.boot.test.system.OutputCaptureExtension;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+
+@ExtendWith({SpringExtension.class, MockitoExtension.class, OutputCaptureExtension.class})
+@SpringBootTest
+@ActiveProfiles("noLogTest")
+class AutoBumpServiceLoggerTest {
+    @Autowired
+    private AtlassianHostRepository repository;
+
+    @Mock
+    private Autobumpconfig autobumpconfig;
+
+    @Autowired
+    @InjectMocks
+    private AutoBumpService testService;
+
+    @BeforeEach
+    void setUp() {
+        var provider = Mockito.mock(GitProvider.class);
```

Comment:
I'd suggest to uniformily mock objects by using the @Mock annotation.

@Mock
GitProvider provider; 

instead of:  var provider = Mockito.mock(GitProvider.class);